agent/spec/lib/kontena/workers/fluentd_worker_spec.rb

```diff
@@ -60,7 +60,9 @@
       fluentd = instance_double(Fluent::Logger::FluentLogger)
       subject.wrapped_object.instance_variable_set('@fluentd', fluentd)
       subject.wrapped_object.instance_variable_set('@forwarding', true)
-      expect(fluentd).to receive(:post).with('web.nginx.1', {log: 'foo bar', source: 'stdout'})
+      subject.wrapped_object.instance_variable_set('@node', 'node-1')
```

Comment:
Just a reminder that `instance_variable_set` is not very good practice.

Reply:
yes, not the best way to do things. I'll see if I can make it work some other way around.